loggibud/v1/data_conversion.py

```diff
@@ -3,84 +3,62 @@
 """
 
 from typing import Optional
+from dataclasses import dataclass
 
+import tsplib95
 import numpy as np
 
-from loggibud.v1.distances import calculate_distance_matrix_m
-from loggibud.v1.types import CVRPInstance
+from loggibud.v1.distances import calculate_distance_matrix_m, OSRMConfig
+from loggibud.v1.types import CVRPInstance, JSONDataclassMixin
 
 
-def to_tsplib(
-    instance: CVRPInstance, file_name: Optional[str] = None
-) -> Optional[str]:
-    """Convert instance into TSPLIB format
-
-    Parameters
-    ----------
-    instance
-        CVRP instance
-
-    file_name
-        File with extension ".vrp" to save the instance. If not provided,
-        return a string with the same data
-
-    Returns
-    -------
-    A string with instance data in the TSPLIB format if no ``file_name`` was
-    especified, or nothing otherwise
+@dataclass
+class TSPLIBConversionParams(JSONDataclassMixin):
+
+    osrm_config: Optional[OSRMConfig] = None
+    """Config for calling OSRM distance service."""
+
+    distance_scaling_factor = 10
```

Comment:
Nitpick: add `int` type hint:

```suggestion
    distance_scaling_factor: int = 10
```